pulseapi/profiles/test_views.py

```diff
@@ -478,8 +479,6 @@ def test_profile_list_filtering_active_v2(self):
         self.run_test_profile_list(
             api_version=settings.API_VERSIONS['version_2'],
             profile_serializer_class=UserProfilePublicSerializer,
-            profile_params={'is_active': True},
```

Comment:
Since we are already filtering the QS to return only active profiles, I believe we can remove these!